schema/tags.js

```diff
@@ -0,0 +1,23 @@
+import { gql } from "apollo-server";
+import expensetags from "../dummy-data/expense-tags.js";
+
+const typeDef = gql`
+  extend type Query {
+    tags: [Tag]
+  }
+
+  type Tag {
+    accountId: String
+    name: String
+  }
+`;
+
+const tagResolvers = {
+  Query: {
+    tags: (root, {}, { v1AccessToken, v2AccessToken, dataSources }) => {
+      return { expensetags };
```

Comment:
remove the curly bracket to fix the error you are getting as the schema expects an array to be returned